tools/code_format/check_format.py

```diff
@@ -1110,7 +1059,7 @@ def ownedDirectories(error_messages):
     owned = []
     maintainers = [
         '@mattklein123', '@htuch', '@alyssawilk', '@zuercher', '@lizan', '@snowp', '@asraa',
-        '@yavlasov', '@junr03', '@dio', '@jmarantz'
+        '@yavlasov', '@junr03', '@dio', '@jmarantz', '@antoniovicente'
```

Comment:
Thanks for this update. For review benefit, are there any other sites that aren't refactors or CLI plumbing related?

Reply:
No worries, the diff is hard to read.

I defined a FormatChecker class and moved all the functions in https://github.com/envoyproxy/envoy/blob/130c7c4e271fe306ae4dd747daa5f09ff31aef79/tools/code_format/check_format.py#L186-L1030 under it. I moved these arguments https://github.com/envoyproxy/envoy/blob/130c7c4e271fe306ae4dd747daa5f09ff31aef79/tools/code_format/check_format.py#L1083-L1098 to the initializer.

That particular highlighted change to the owners list is possibly due to my needing to rebase?